### ISSUE TITLE
Nothing happend

### ISSUE THREAD

**Issue by lucasamp** [0XLkjils/PUBG-cheat-hack-undetected-free-2023-#1]
Hello,

I downloaded your archive, there is only one exec file, when i try to open, is nothing happend. In game at least

**Comment by 0XLkjils** [comment]
Hello there, did you turn off antivirus? It wont work on virtual machine
If still not working try to restart your computer or delete and download your game. After this all issues should be gone
Thanks!

**Comment by lucasamp** [comment]
> Hello there, did you turn off antivirus? It wont work on virtual machine If still not working try to restart your computer or delete and download your game. After this all issues should be gone Thanks!

Yep, i restart the computer, no virtual machine, nothing, happen's. I try this things more times.
